build-tools/installers/create-installers.targets

```
@@ -0,0 +1,122 @@
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Options.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Terminal.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\monodoc.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\MULTIDEX_JAR_LICENSE"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\**\proguard.jar"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\**\license.html"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\r8.jar"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\SgmlReaderDll.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin*"></MSBuildFiles>
+  </ItemGroup>
+  <ItemGroup> <!-- monodroid -->
+    <MSBuildFiles Condition="Exists('$(MSBuildSrcDir)\ICSharpCode.SharpZipLib.dll')" Include="$(MSBuildSrcDir)\ICSharpCode.SharpZipLib.dll"></MSBuildFiles>
+    <MSBuildFiles Condition="Exists('$(MSBuildSrcDir)\INIFileParser.dll')" Include="$(MSBuildSrcDir)\INIFileParser.dll"></MSBuildFiles>
+    <MSBuildFiles Condition="Exists('$(MSBuildSrcDir)\jar2xml.jar')" Include="$(MSBuildSrcDir)\jar2xml.jar"></MSBuildFiles>
+    <MSBuildFiles Condition="Exists('$(MSBuildSrcDir)\Mono.AndroidTools.dll')" Include="$(MSBuildSrcDir)\Mono.AndroidTools.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Android.DebugRuntime-*"/>
+  </ItemGroup>
+  <ItemGroup>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-arm.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-arm64.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-x86.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-x86_64.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\opt.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\llc.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\**\proguard.bat"></MSBuildFilesWin>
+  </ItemGroup>
+  <ItemGroup>
+    <MSBuildHostFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\cross-arm"></MSBuildHostFilesUnix>
+    <MSBuildHostFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\cross-arm64"></MSBuildHostFilesUnix>
+    <MSBuildHostFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\cross-x86"></MSBuildHostFilesUnix>
+    <MSBuildHostFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\cross-x86_64"></MSBuildHostFilesUnix>
+    <MSBuildHostFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\illinkanalyzer"></MSBuildHostFilesUnix>
+    <MSBuildHostFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\mono"></MSBuildHostFilesUnix>
+    <MSBuildHostFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\mono-symbolicate"></MSBuildHostFilesUnix>
+    <MSBuildHostFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\opt"></MSBuildHostFilesUnix>
+    <MSBuildHostFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\llc"></MSBuildHostFilesUnix>
+    <MSBuildFilesUnix Include="$(MSBuildSrcDir)\**\proguard.sh"></MSBuildFilesUnix>
```

Similarly, this should avoid the `**` wildcard and specify the path explicitly:

```suggestion
    <MSBuildFilesWin Include="$(MSBuildSrcDir)\proguard\bin\proguard.sh" />
```